### DISCUSSION TITLE
Add sponsorship ability

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#3208] [Ideas]
Inspired by the combine talk on open worm, which is able to maintain some level of  funding through donations. 
here is there open collective: https://opencollective.com/openworm

We can set up an open collective to manage the legal side of having an organization etc. The open worm project actually chose to form on own non profit organization, which is probably a bit premature for us. 

@jonrkarr @hsauro @moraru 
Do you have any thoughts on signing up with open collective?

**Comment by jonrkarr** [comment]
I'm not familiar with the administrative side of how this intersects with us working for universities, unlike OpenWorm. @moraru @hsauro have you dealt with this?

One option would be to post information about donations and deal with managing it when someone actually tries to make a donation.

**Comment by jonrkarr** [comment]
OpenWorm is an interesting counterpoint to the the predominant grant-funded model of research.

**Comment by bilalshaikh42** [comment]
@jonrkarr 

Another aspect to keep in mind is trademark concerns with our name/logo etc.  With some sort of legal entity, we could register trademarks to prevent any issues with some claims on the term BioSimulation. 

I think switching our licence to Apache 2.0 would be a good start, as it includes language re trademarks etc
https://google.github.io/opencasebook/trademarks/#license-terms-bearing-on-trademark-use
